measuring-client/src/main/java/de/cyface/app/ui/nav/view/CursorMeasureAdapter.java

```diff
@@ -85,40 +96,61 @@ public View newView(final Context context, final Cursor cursor, final ViewGroup
-        String label = "(" + measurementId + ") " + dateText + " (" + distanceText + ") - "
-                + getTranslation(contextWeakReference, modality);
-
-        // Disable synced and non-finished items to disallow deletion
-        final MeasurementStatus status = MeasurementStatus
-                .valueOf(cursor.getString(cursor.getColumnIndex(MeasurementTable.COLUMN_STATUS)));
-        if (status == MeasurementStatus.OPEN || status == MeasurementStatus.PAUSED
-                || status == MeasurementStatus.SYNCED) {
-            label += " - " + status.toString().toLowerCase();
+        try {
+            final TextView itemView = view.findViewById(R.id.data_row_text);
+            final int measurementId = cursor.getInt(cursor.getColumnIndex(BaseColumns._ID));
+
+            // Measurement time
+            final long measurementTimestamp = cursor.getLong(cursor.getColumnIndex(MeasurementTable.COLUMN_TIMESTAMP));
+            final Date date = new Date(measurementTimestamp);
+            final String dateText = new SimpleDateFormat("dd.MM.yy HH:mm", Locale.GERMANY).format(date);
+
+            // Retrieve distance data
+            final double distance = cursor.getDouble(cursor.getColumnIndex(MeasurementTable.COLUMN_DISTANCE));
+            final int distanceMeter = (int) Math.round(distance);
+            final double distanceKm = distanceMeter == 0 ? 0.0 : distanceMeter / 1000.0;
+            final String distanceText = Math.round(distanceKm * 10) / 10.0 + " km";
+
+            // Measurement duration
+            //TODO: After adding measurement statistic view, only show duration there (performance)
+            final long durationMillis = persistenceLayer.loadDuration(measurementId);
+            final long durationSeconds = durationMillis / 1000;
+            final long durationMinutes = durationSeconds / 60;
+            final long durationHours = durationMinutes / 60;
+            final String hoursText = durationHours > 0 ? durationHours + "h " : "";
+            final String minutesText = durationMinutes > 0 ? durationMinutes % 60 + "m " : "";
+            final String secondsText = durationSeconds % 60 + "s";
+            final String durationText = hoursText + minutesText + secondsText;
+
+            // Average speed
+            //TODO: After adding measurement statistic view, only show speed there (performance)
+            final double averageSpeedMps = persistenceLayer.loadAverageSpeed(measurementId, new DefaultLocationCleaningStrategy());
+            final double averageSpeedKmh = averageSpeedMps * 3.6;
+            final String averageSpeedText = Math.round(averageSpeedKmh) + " km/h";
+
+            // Retrieve modality
+            final Modality modality = Modality
+                    .valueOf(cursor.getString(cursor.getColumnIndex(MeasurementTable.COLUMN_MODALITY)));
+
+            // Set List Item Text
+            String label = measurementId + ") " + dateText + " (" + distanceText + " "+ durationText +") "
```

Comment:
18) 08.01.23 17:58 (0.0 km 1m 32s) Fahrrad 0 km/h [- open/paused/synced/skipped]

It's just for testing purposes right now as there is no metrics view for finished measurements.